### PR TITLE
My created project

### PR DESCRIPTION
**Pull request by JeromeJULIEN**

### CODE REVIEW

File: web/components/befundrProgram/befundr-project-access.tsx [hunk 1]
```diff
@@ -40,6 +40,54 @@ export function useBefundrProgramProject() {
...
+        const projects = [];
+        // loop to get all project
+        for (let i = 0; i < createdProjectCounter; i++) {
+          try {
+            // Generate the seed
+            const [projectPDA] = await PublicKey.findProgramAddress(
+              [
+                Buffer.from('project'),
+                userPublicKey.toBuffer(),
+                new BN(i + 1).toArray('le', 2),
```
Comment (JonathanDugard) [inline web/components/befundrProgram/befundr-project-access.tsx:65]: created_project_counter is a u16 then I don't think it has to be converted in BN (TBC)

File: web/components/befundrProgram/befundr-project-access.tsx [hunk 2]
```diff
@@ -40,6 +40,54 @@ export function useBefundrProgramProject() {
...
+
+            // push the projects to projects
+            if (projectData) {
+              projects.push({
+                publicKey: projectPDA,
+                account: projectData,
+              });
+            }
+          } catch (error) {
+            console.error(`Error fetching project ${i}:`, error);
```
Comment (JonathanDugard) [inline web/components/befundrProgram/befundr-project-access.tsx:81]: Would be great to indicate the public key instead of the index (or in addition to it)

**Review by JonathanDugard — APPROVED** [review]

File: web/components/befundrProgram/befundr-project-access.tsx (hunk 2 above)
Comment (JeromeJULIEN) [reply web/components/befundrProgram/befundr-project-access.tsx:81]: the PDA is displayed in the error

File: web/components/befundrProgram/befundr-project-access.tsx [hunk 3]
```diff
@@ -40,6 +40,54 @@ export function useBefundrProgramProject() {
     });
   };
 
+  //* fetch all project created by a user
+  const getAllCreatedProjectsByAUser = (
```
[the remark this answers is not in the document]
Comment (JeromeJULIEN) [reply web/components/befundrProgram/befundr-project-access.tsx]: done

**Pull request merged**